deploy/kubernetes-the-hard-way.md

```diff
@@ -0,0 +1,47 @@
+# Kubernetes The Hard Way
+
+---
+此文件翻译来源:[kubernetes-the-hard-way](https://github.com/kelseyhightower/kubernetes-the-hard-way)
+
+如有翻译不好的地方或文字上的错误, 都欢迎提出 issue 或是 PR
+
+繁中版: [Kubernetes-The-Hard-Way-ZH-tw](https://github.com/kweisamx/Kubernetes-The-Hard-Way-ZH-tw)
+
+---
+
+这份教学将带领你走上安装kubernetes的艰辛之路。这份文件不适用于想要一键自动化部属kubernetes丛集的人。如果你想要轻松部属, 可以参考[Google Container Engine](https://cloud.google.com/container-engine) 或[Getting Started Guides](http://kubernetes.io/docs/getting-started-guides/)
+
+Kubernetes The Hard Way 是个学习的最佳方式, 会花上许多时间确保你真正了解每项组件的任务以及需求,去搭建整个kubernetes 丛集
+> 这份文件不应该出现在production 文件中, 也无获得kubernetes社区的许多支持, 但这都不影响你想真正了解Kubernetes的决心！
+
+---
+
+## 谁适合看这份文件？
+这份文件的目标是给那些计画要使用kubernetes 当作production环境的人, 并想了解每个有关kubernetes的环节以及他们如何运作的
+
+## 有关丛集的详细资讯
+Kubernetes The Hard Way 将引导你建立高可用的Kubernetes的丛集, 包括每个组件之间的加密以及RBAC认证
+
+* [Kubernetes](https://github.com/kubernetes/kubernetes) 1.8.0
+* [cri-containerd Container Runtime](https://github.com/kubernetes-incubator/cri-containerd) 1.0.0-alpha.0
+* [CNI Container Networking](https://github.com/containernetworking/cni) 0.6.0
+* [etcd](https://github.com/coreos/etcd) 3.2.8
+
+## 实验步骤
+
+这份教学假设你已经有办法登入[Google Cloud Platform](https://cloud.google.com), GCP被用来作为这篇教学的基础需求,你也可以将这篇教学应用在其他平台上
+
+* [事前準备](docs/01-prerequisites.md)
```

Comment:
Those markdown files are missing?

Reply:
Oh, That my wrong ,
Thak OK, I will update later